src/Controller/FestivEllesController.php

```diff
@@ -23,9 +24,15 @@ public function index(ArticleRepository $articleRepository, CalendarRepository $
             ['type' => 'festivelles'],
             ['date' => 'ASC']
         );
+        $members = $this->getDoctrine()->getRepository(Member::class)->findBy([], ["title" => "ASC"]);
```

Comment:
According to symfony best practices, Repositories (and Services in general) should be get using dependency injection